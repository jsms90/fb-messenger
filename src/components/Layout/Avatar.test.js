import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
// import Adapter from 'enzyme-adapter-react-16'
import Avatar  from './Avatar'

// TASK 1: configure the Enzyme adapter. Hint https://airbnb.io/enzyme/
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('<Avatar />', () => {
  it('renders Avatar', () => {
    // You can use console.log(wrapper.debug()) to console.log the component that you are testing
    // TASK 2: Shallow the Avatar component
    //    Hint:  https://github.com/adriantoine/enzyme-to-json#helper
    // TASK 3: Create the snapshot
    const wrapper = shallow(<Avatar username="my-lovely-username"/>)
    console.log(wrapper.debug())
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
