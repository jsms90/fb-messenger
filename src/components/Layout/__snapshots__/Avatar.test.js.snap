// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> renders Avatar 1`] = `
<styled.img
  alt="my-lovely-username"
  size="medium"
  src="/images/my-lovely-username_lg.jpg"
/>
`;
